.github/workflows/benchmark-command.yml

```diff
@@ -89,9 +89,15 @@ jobs:
           role-to-assume: arn:aws:iam::369469875935:role/AutoGluonCIBenchmarkConfig
           role-duration-seconds: 3600
           aws-region: us-east-1
+      - name: Extract Fork Info
+        id: parse_fork_info
+        run: |
+          IFS="|" read -r fork_name fork_branch <<< "${{ github.event.inputs.fork_info }}"
```

Comment:
Though `fork_name` and `fork_branch` are not directly run as a script in this step, please make sure they are not being used for script execution in subsequent steps to precent the risk of [script injection](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections)

Reply:
The benchmark workflow from master is run and that too only by maintainers thus preventing this. 
But still made changes and used the env method to mitigate this.